src/js/components/deployments/deployment-wizard/phasesettings.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import pluralize from 'pluralize';
+
+import Chip from '@material-ui/core/Chip';
+import AddIcon from '@material-ui/icons/Add';
+
+import { Table, TableBody, TableCell, TableHead, TableRow, Select, MenuItem, Input, InputAdornment, IconButton } from '@material-ui/core';
+import CancelIcon from '@material-ui/icons/Cancel';
+import { getRemainderPercent } from '../../../helpers';
+
+export default class PhaseSettings extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+  }
+
+  updateDelay(value, index) {
+    let newPhases = this.props.phases;
+    // value must be at least 1
+    value = value>0 ? value : 1;
+
+    // set value based on unit 
+    value = newPhases[index].delayUnit === 'days' ? value*24 : value;
+    newPhases[index].delay = value;
+
+    this.props.updatePhaseStarts(newPhases);
+    // logic for updating time stamps should be in parent - only change delays here
+  }
+
+  updateBatchSize(value, index) {
+    let newPhases = this.props.phases;
+
+    if (value < 1) {
+      value = 1;
+    } else if (value >100) {
+      value = 100;
+    }
+
+    newPhases[index].batch_size = value;
```

Comment:
this has to be `newPhases[index].batch_size = Number(value);` otherwise the `batch_size` will end up as a string and fail to be parsed in the backend

Reply:
Thanks! Fixed in 0f3daf0